src/BlockTypes/MiniCart.php

```diff
@@ -371,9 +371,7 @@ protected function get_markup( $attributes ) {
 		</span>';
 
 		if ( is_cart() || is_checkout() ) {
-			return '<div class="' . $wrapper_classes . '">
-				<button class="wc-block-mini-cart__button ' . $classes . '" aria-label="' . esc_attr( $aria_label ) . '" style="' . $style . '" disabled>' . $button_html . '</button>
-			</div>';
+			return;
 		}
```

Comment:
If we no longer need to render real cart data here, we can move this check to the top of the `get_markup()` method so we return earlier.

Reply:
> One issue of this approach that I didn't consider initially is that if the Mini Cart block is added to a Row block, then it doesn't occupy any space when it renders and empty markup. For example, see how the menu is right-aligned in the Checkout page but centered in the My Account page.
> 
> Checkout page:
> 
> ![imatge](https://user-images.githubusercontent.com/3616980/156788567-43174beb-d637-4bce-af51-652a20bec5da.png)
> 
> My Account page:
> 
> ![imatge](https://user-images.githubusercontent.com/3616980/156788645-d5e6f8e7-af02-4069-b7a8-2d2a4dc46708.png)
> 
> I wonder if instead of returning empty markup, we should render the same markup but set its `visibility` to hidden and `aria-hidden` to true.
> 
> Thoughts?

Great catch! I never tested the Mini Cart Block in the row block. Thanks for pointing this out! 🙏 

> If we no longer need to render real cart data here, we can move this check to the top of the get_markup() method so we return earlier.

About this, I think that if we want to use this solution it is better to continue to render the entire block (with the classes that come from Global Styles too). If there is a class/style that increases the size of the Mini Cart, and then we don't use them on these two pages we might have a similar problem.

Reply:
> About this, I think that if we want to use this solution it is better to continue to render the entire block (with the classes that come from Global Styles too). If there is a class/style that increases the size of the Mini Cart, and then we don't use them on these two pages we might have a similar problem.

Makes sense. :ok_hand: